--- a/yaml_cmd.py	2020-12-14 22:13:19.000000000 -0500
+++ b/yaml_cmd.py	2020-12-14 22:13:34.067115865 -0500
@@ -513,8 +513,6 @@
     def round_trip_save(self, file_name):
         inp = open(file_name).read()
         backup_file_name = file_name + '.orig'
-        if not os.path.exists(backup_file_name):
-            os.rename(file_name, backup_file_name)
         return self.round_trip_single(inp, out_file=file_name)
 
     def round_trip_input(self, inp):
